src/components/content/credential/Credential.tsx

```diff
@@ -234,7 +230,7 @@ function Credential(): JSX.Element {
                     <UpdateCredential
                         createCredential={createCredential}
                         onUpdateCancel={onUpdateCancel}
-                        getCredentials={getCredentials}
+                        credentialsQuery={credentialsQuery}
```

Comment:
in line 257, can you also please add a 'loading' flag based on the query status.